python/cuml/test/test_random_forest.py

```diff
@@ -152,17 +152,17 @@ def test_rf_regression(datatype, split_algo, mode,
 
 
 @pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('column_info', [unit_param([16, 7]),
+@pytest.mark.parametrize('column_info', [unit_param([20, 7]),
                          quality_param([200, 100]),
                          stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(20), quality_param(5000),
+@pytest.mark.parametrize('nrows', [unit_param(100), quality_param(5000),
```

Comment:
Was this causing an issue for the patch when the `nrows` was set to 20?

Reply:
yes, opened an issue #1303 and #1306 that @Salonijain27 is looking into